app/components/RoomList/index.js

```diff
@@ -202,12 +203,18 @@ class RoomList extends Component {
 	}
 
 	renderRooms = () => {
-		let { eqpmtInRooms, roomList, historyPerRoom, roomFilter } = this.state
+		let { eqpmtInRooms, roomList, historyPerRoom, roomFilter, statusFilter } = this.state
 
+		// filter by rid
 		roomList = roomList.filter(({ rid }) =>
 			rid.toLowerCase().indexOf(roomFilter.toLowerCase()) > -1
 		)
 
+		// filter by rstatus
+		roomList = roomList.filter(room =>
+			statusFilter.includes(room.rstatus)
```

Comment:
if the `statusFilter` is going to be blank by default, let's only filter if `statusFilter` has a value